Employee.java

```diff
@@ -0,0 +1,15 @@
+public class Employee {
+    private String fullName;
+    private String position;
+    private String email;
+    private String phone;
+    private int age;
+
+    public Employee(String fullName, String position, String email, String phone, int age) {
+        this.fullName = fullName;
+        this.position = position;
+        this.email = email;
+        this.phone = phone;
+        this.age = age;
+    }

```

Comment:
Incapsulation consists of access modifiers and getters + setters. Please add getters and setters for all fields.